coding_conventions.tex

```diff
@@ -280,15 +290,17 @@ \section{Code}
 \subsection{General}
 %
 \begin{itemize}
-	\item Line length should not exceed 80 characters
+	\item Line length should not exceed 120 characters
 %
-	\item Use \mintinline{text}{camelCase} for variable/string/wave/dfref names and \mintinline{text}{CamelCase} for structures
+	\item Use \mintinline{text}{camelCase} for variable/string/wave/dfref names and \mintinline{text}{CamelCase} for functions and structures
 %
 	\item Prefer structure-based GUI control procedures over old-style functions
 %
 	\item The variables \mintinline{text}{i, j, k, l} are reserved for loop counters, from outer to inner loops
 %
 	\item Use free waves for temporary waves
+%
+	\item Prefer generic builtin functions like \mintinline{igor}{IndexToScale, DimSize} over their 1D counterparts \mintinline{igor}{IndexToScale, DimSize}.
```

Comment:
Did you mean as 1D counterparts "pnt2x" and "numpnts"?

Reply:
Thx, fixed.